environments/.stackhpc/hooks/post-bootstrap.yml

```diff
@@ -3,14 +3,15 @@
   gather_facts: false
   tags: podman
   tasks:
-    - name: Configure container image registry for unqualified searches to avoid docker.io ratelimits
+    - name: Configure container image registry to avoid docker.io ratelimits
```

Comment:
Does this get used in image-build anywhere?

If so, should be removed in `post.yml`.

Reply:
No, because the hosts: for this play are `podman:!builder`